.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please use linter for HTML, CSS, and JavaScript from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config.